src/module.cc

```diff
@@ -1,5 +1,6 @@
-#include "node_report.h"
+// Copyright 2017 Nodereport
```

Comment:
I'll et @jasnell confirm but I think it should be:

```
// Copyright node-report contributors. All rights reserved.
// SPDX-License-Identifier: MIT
```

in line with https://github.com/nodejs/node/pull/10599/files as these are "new" files that were never under the joyent copyright.